api.bs

```diff
@@ -1110,6 +1114,49 @@ if the user has opted out of collection of diagnostic data.
 * User ability to view the impression store and past report submissions.
 
 
+# HTTP API # {#http-api}
+
+\`<dfn http-header><code>Save-Impression</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=PrivateAttribution>saveImpression()</a> API.
+
+<pre class=example id=ex-save-impression-header>
+Save-Impression: conversion-site=advertiser.example, histogram-index=2, \
+  filter-data=12, lifetime-days=7
+</pre>
+
+The following keys are defined, corresponding to the members of
+the {{PrivateAttributionImpressionOptions}} dictionary passed to
+<a method for=PrivateAttribution>saveImpression()</a>.
+
+<dl dfn-for=save-impression>
+  <dt><code>conversion-site</code></dt>
+  <dd>
+    Value of <a dict-member for=PrivateAttributionImpressionOptions>conversionSite</a>,
+    a [=structured header/string=]. This key is required.
```

Comment:
This is listed as a string, but the example above is encoded as a token. (Note the lack of surrounding `"`.)

@martinthomson Do you think we should bother with string here, or even support both?

Reply:
Yeah, I think that we can probably support tokens, but the question is whether we would want to.  We're going to have to deal with IDNA.  Punycode-encoded IDNA will work as a token.  The only reason to stick with just string is to avoid the unnecessary flexibility.

We should be very careful when specifying this, so I would say that we're not done either way.  We need to be very clear that this is punycode-encoded only and that it is not valid to include Unicode, even using the percent-encoded display string type.

Reply:
```suggestion
    a [=structured header/string=].
    The string value includes a domain name using A-labels only;
    [[RFC5890|Internationalized Domain Names]] therefore need to use [[RFC3492|punycode]]. 
    This key is required.
```

Note that this last line is not 100% consistent with the changes in #139.  We'll need to resolve that in #145.